### ISSUE TITLE
Error 1406: Data too long for column 'payload_content'

### ISSUE THREAD

**Issue by pboguslawski** [go-gitea/gitea#19679]
### Description

Noticed one occurrence of error

```
2022/05/09 11:20:49 ...n/webhook/webhook.go:587:NotifyPushCommits() [E] PrepareWebhooks: CreateHookTask: Error 1406: Data too long for column 'payload_content' at row 1
```
in gitea log.

### Gitea Version

1.16.3

### Can you reproduce the bug on the Gitea demo site?

No

### Git Version

2.30.2

### Operating System

Debian

### How are you running Gitea?

Gitea build from sources using go 1.17.9.

### Database

MySQL

**Comment by wxiaoguang** [comment]
Similar to this one:
* https://github.com/go-gitea/gitea/issues/16656
* https://github.com/go-gitea/gitea/pull/16765

MySQL's TEXT can only hold 64K string. It should be altered to LONGTEXT if you have a lot of data.

**Comment by 42wim** [comment]
Fixed in #20038
